app/jobs/regular/ensure_badge_consistency.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Jobs
+  class EnsureBadgeConsistency < ::Jobs::Base
```

Comment:
Should I add this here?

```
sidekiq_options queue: "low"
```

Reply:
Is this job blocking the default queue frequently? Otherwise, I wouldn't worry about the queue it is in for now.

Reply:
The old job was running for more than 2h. I think a considerable amount of the time was spent doing this, so yes, I believe it may block a thread of a Sidekiq process until the job is over. However, I do not think our queue handling is optimal right now so this line might not have the desired effect.